python/TestHarness/testers/RunCommand.py

```diff
@@ -0,0 +1,30 @@
+import re, os, sys, time
+from Tester import Tester
+from RunParallel import RunParallel # For TIMEOUT value
+
+class RunCommand(Tester):
+
+  @staticmethod
+  def validParams():
+    params = Tester.validParams()
+    params.addRequiredParam('command',      "The command line to execute for this test.")
```

Comment:
Well, the good way (or at least "industry standard") is containers. But it apparently has not been an issue... yet...